__tests__/app.test.js

```diff
@@ -70,6 +70,7 @@ describe("GET /api/articles/:article_id", () => {
           votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: "11",
```

Comment:
Rather than amending an old test, we should have a new test to show that comment_count now exists. We shouldn't amend any older tests after we've moved on. If the test is failing, have a think about how the test could be made to be more flexible 👍 🙂